homeassistant/components/broadlink/remote.py

```diff
@@ -83,6 +85,7 @@
     """Set up the Broadlink remote."""
     host = config[CONF_HOST]
     mac_addr = config[CONF_MAC]
+    devtype = config[CONF_TYPE]
```

Comment:
Please use snake_case for all variables.
```suggestion
    dev_type = config[CONF_TYPE]
```